src/main/java/info/matai/services/plugins/PluginFileReaderService.java

```diff
@@ -1,14 +1,40 @@
 package info.matai.services.plugins;
 
-import info.matai.models.configuration.Plugin;
-import java.util.ArrayList;
-import java.util.List;
+import info.matai.models.configuration.Server;
+import info.matai.models.configuration.ServerLock;
+import info.matai.services.common.utils.FileUtils;
+import info.matai.services.common.utils.JsonUtils;
 
-class PluginFileReaderService {
+import java.io.File;
+import java.util.logging.Logger;
 
-  public List<Plugin> getPlugins() {
-    final List<Plugin> plugins = new ArrayList<Plugin>();
+public class PluginFileReaderService {
 
-    return plugins;
-  }
+    private static final Logger LOGGER = Logger.getGlobal();
```

Comment:
pourquoi un par fichier et pas juste un pour toute l'app?

Reply:
Ça nous permet de mieux sa voir d'où vient l'erreur:
```
avr. 05, 2022 1:13:35 AM info.matai.services.plugins.PluginFileReaderService getServerFromPlugin
INFOS: Everything is OK
```
vs
`INFO: Everything is OK` 
OU
```
avr. 05, 2022 1:15:59 AM info.matai.services.Logging info
INFOS: Everything is OK
```

Malgré qu'à voir le 2e format (`INFO: Everything is OK` ), je serais pas trop contre ça vu que ça serait principalement du output pour le user, t'en penses quoi?